doc/tutorials/tutorials.txt

```diff
@@ -39,6 +39,7 @@ progress to complex, extensible uses.
  - \ref cftutorial
  - \ref akfntutorial
  - \ref cnetutorial
+ - \ref optimizertutorial
```

Comment:
This might be better as part of the 'policy class documentation'?  Not sure, up to you.

Reply:
Sounds good to me.